### PR TITLE
adding tomato checkbox

### PR DESCRIPTION
**Pull request by pjaudiomv**
this adds a checkbox that will autofill the tomato url

### CODE REVIEW

**Comment by pjaudiomv** [comment]
Screen Shot 2020-01-09 at 5 15 34 PM

File: js/bmlt_tabs_admin.js [hunk 1]
```diff
@@ -36,6 +37,7 @@ function numbersonly(myfield, e, dec)
 };
 
 jQuery(document).ready(function($) {
+	var tomato = "https://tomato.na-bmlt.org/main_server";
```
Comment (radius314) [inline js/bmlt_tabs_admin.js]: can you set to `https://tomato.bmltenabled.org/main_server`

File: readme.txt [hunk 2]
```diff
@@ -36,6 +36,9 @@ Crouton was forked from BMLT Tabbed UI plugin in 2018.  This plugin provides a T
 
 == Changelog ==
 
+= 3.5.1 =
+* Added checkbox for using tomato as root server. [#181]
+
```
Comment (radius314) [inline readme.txt]: 3.5.0 isn't released yet.  Can you adjust?

File: crouton.php [hunk 3]
```diff
@@ -427,6 +427,10 @@ public function adminOptionsPage()
                                 <label for="root_server">Default Root Server: </label>
                                 <input id="root_server" type="text" size="50" name="root_server" value="<?php echo $this->options['root_server']; ?>" /> <?php echo $connect; ?>
                             </li>
+                            <li>
+                                <input type="checkbox" id="use_tomato" name="use_tomato" value="1"/>
+                                <label for="use_tomato">Use Tomato &#127813; (Root Server Aggregator)</label>
```
Comment (radius314) [inline crouton.php]: Might be helpful to have a help tooltip explaining what 🍅 is.

File: js/bmlt_tabs_admin.js (hunk 1 above)
Comment (pjaudiomv) [reply js/bmlt_tabs_admin.js]: nice will do, this is exact reason I wanted to add this feature. I cant even remember tomato url how are other people.

File: readme.txt (hunk 2 above)
Comment (pjaudiomv) [reply readme.txt]: oops sure thing

File: crouton.php (hunk 3 above)
Comment (pjaudiomv) [reply crouton.php]: true true

**Comment by pjaudiomv** [comment]
Screen Shot 2020-01-09 at 7 13 41 PM

**Pull request merged**